dataverse-webapp/src/main/resources/socialscience_en.properties

```diff
@@ -1,85 +1,373 @@
+datasetfieldtype.targetSampleActualSize.withParent.title=Targeted Sample Size
+datasetfieldtype.targetSampleSizeAchieved.title=Achieved
+datasetfieldtype.targetSampleSizeAchieved.withParent.title=Achieved Sample Size
+datasetfieldtype.deviationsFromSampleDesign.title=Sample Deviations
+datasetfieldtype.substitutionOfRespondents.title=Substitution of Respondents
+datasetfieldtype.substitutionOfRespondentsAcceptability.title=Acceptability
+datasetfieldtype.substitutionOfRespondentsAcceptability.withParent.title=Substitution of Respondents - Acceptability
+datasetfieldtype.substitutionOfRespondentsDescription.title=Description
+datasetfieldtype.substitutionOfRespondentsDescription.withParent.title=Substitution of Respondents - Description
+datasetfieldtype.ageCutOff.title=Age Cut-Off
+datasetfieldtype.ageCutOffLower.title=Lower
+datasetfieldtype.ageCutOffLower.withParent.title=Lower Age Cut-Off
+datasetfieldtype.ageCutOffUpper.title=Upper
+datasetfieldtype.ageCutOffUpper.withParent.title=Upper Age Cut-Off
 datasetfieldtype.collectionMode.title=Collection Mode
+datasetfieldtype.contactAttempts.title=Contact Attempts
+datasetfieldtype.contactAttemptsNumber.title=Required Number of Contact Attempts
+datasetfieldtype.contactAttemptsNumber.withParent.title=Required Number of Contact Attempts
+datasetfieldtype.contactAttemptsSameDay.title=Contact attempts at different times of day required
+datasetfieldtype.contactAttemptsSameDay.withParent.title=Contact Attempts at Different Times of Day Required
+datasetfieldtype.contactAttemptsDifferentDays.title=Contact attempts on different days of week required
+datasetfieldtype.contactAttemptsDifferentDays.withParent.title=Contact Attempts on Different Days of Week Required
 datasetfieldtype.researchInstrument.title=Type of Research Instrument
 datasetfieldtype.dataCollectionSituation.title=Characteristics of Data Collection Situation
+datasetfieldtype.weighting.title=Weight
 datasetfieldtype.actionsToMinimizeLoss.title=Actions to Minimize Losses
 datasetfieldtype.controlOperations.title=Control Operations
-datasetfieldtype.weighting.title=Weighting
+datasetfieldtype.interviewsBackcheck.title=Interviews' Back-Check
+datasetfieldtype.interviewsBackcheckPercentage.title=Percentage of Interviews
+datasetfieldtype.interviewsBackcheckPercentage.withParent.title=Percentage of Back-Checked Interviews
+datasetfieldtype.interviewsBackcheckProcedureDescription.title=Back-Check Procedure Description
+datasetfieldtype.interviewsBackcheckProcedureDescription.withParent.title=Descritpion of Back-Check Procedure
 datasetfieldtype.cleaningOperations.title=Cleaning Operations
 datasetfieldtype.datasetLevelErrorNotes.title=Study Level Error Notes
 datasetfieldtype.responseRate.title=Response Rate
 datasetfieldtype.samplingErrorEstimates.title=Estimates of Sampling Error
-datasetfieldtype.otherDataAppraisal.title=Other Forms of Data Appraisal
+datasetfieldtype.otherDataAppraisal.title=otherDataAppraisal
```

Comment:
Is this title correct?

Reply:
No - It shouldn't be the same as original